content/en/docs/topics/kubernetes_apis.md

```diff
@@ -0,0 +1,127 @@
+
+> Note: In all cases of updating a Helm release with supported APIs, you should
+never rollback the release to a version prior to the release version with the
+supported APIs.
+
+If you don't update a release as suggested previously, you will have an error
+similar to the following when trying to upgrade a release in a Kubernetes version
+where its API version(s) is/are removed:
+
+```
+Error: UPGRADE FAILED: unable to build kubernetes objects from current release
+manifest: unable to recognize "": no matches for kind "Deployment" in version
+"apps/v1beta1"
+```
+
+Helm fails in this scenario because it attempts to create a diff patch between
+the current deployed release (which contains the Kubernetes APIs that are removed
+in this Kubernetes version) against the chart you are passing with the
+updated/supported API versions. The underlying reason for failure is that when
+Kubernetes removes an API version, the Kubernetes Go client library can no longer
+parse the deprecated objects and Helm therefore fails calling the library. Helm
+unfortunately is unable to recover from this situation and is no longer able to
+manage such a release.
+See [Updating API Versions of a Release Manifest](#updating-api-versions-of-a-release-manifest)
+for more details on how to recover from this scenario.
+
+## Updating API Versions of a Release Manifest
+
+The manifest is a property of the Helm release object which is stored in the data
+field of a Secret (default) or ConfigMap in the cluster. The data field contains
+a gzipped object which is base 64 encoded twice. There is a Secret/ConfigMap per
+release version/revision in the namespace of the release.
+
+Here are steps to perform an update of the API versions of a release manifest:
+
+- Get latest deployed release:
+`kubectl get secret <release_version_secret_name> -n <release_version_namespace> -o yaml > release.bak`
+- Decode the release object: 
+`cat release.bak | grep -oP '(?<=release: ).*' | base64 -d | base64 -d | gzip -d - > release.data.decoded`
+- Change API versions of the manifests. Can use any tool (e.g. editor) to make
+the changes. This is in the `manifest` field of your decoded release
+object (`release.data.decoded`)
+- Encode the release object: `cat release.data.decoded | gzip | base64 | base64`
+- Replace `data.release` property value in the deployed release file (`release.bak`)
+with the new encoded release object
+- Apply file to namespace: `kubectl apply -f release.bak -n <release_version_namespace>`
+- Perform a `helm upgrade` with a version of the chart with supported Kubernetes
+API versions
+- Add a description in the upgrade, something along the lines to not perform a
+rollback to a Helm version prior to this current version
+
+Alternatively, you can use the Helm [mapkubeapis](https://github.com/hickeyma/helm-mapkubeapis)
+plugin. It performs the update of a release to supported APIs automatically.
+Check out the readme for more details. 
```

Comment:
I would mention your plugin before mentioning the actual detailed steps, so the users don't get all worried about having to do complex manual operations.  After that we can mention that "Alternatively, these are the steps to update..."

Reply:
ok, changed positions.